docs/troubleshooting.md

```diff
@@ -122,9 +122,63 @@ version and build numbers in order to target patches at specific releases. You
 can fix this problem by either setting `manageAppVersionAndBuildNumber` to false
 or removing the value from your export options .plist file.
 
+## I see a `The release artifact contains native changes` warning when running `shorebird release` (#native-changes)
+
+The `shorebird patch` command will print a warning if it detects changes to
+files in your compiled app that correspond to native code changes (`.dex` files
+on Android, files in the `Runner.app` directory on iOS). This does not always
+mean that your patch will not work, but because shorebird cannot be sure that
+the changes are safe, and because shorebird can't patch non-Dart code, it prints
+a warning.
```

Comment:
This seems like it's in the wrong section – the warning below is about asset changes, not native changes

Reply:
I think the warning says "native changes"

Reply:
Fixed.